components/Icons/index.tsx

```diff
@@ -35,3 +39,112 @@ export const DownIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElem
+        <path d="M29.1313 34.2324C36.7945 35.5106 43.4682 33.7812 44.0372 30.3696C44.6063 26.9581 38.8553 23.1562 31.192 21.878C23.5287 20.5997 16.8551 22.3291 16.286 25.7407C15.717 29.1523 21.468 32.9541 29.1313 34.2324Z" fill="#FF5E5E" />
+      </g>
+      <defs>
+        <linearGradient id="paint0_linear_1022_31454" x1="-1.94444" y1="30.5998" x2="59.1667" y2="30.5998" gradientUnits="userSpaceOnUse">
+          <stop stopColor="white" stopOpacity="0" />
+          <stop offset="0.239583" stopColor="white" stopOpacity="0.6" />
+          <stop offset="0.5" stopColor="white" />
+          <stop offset="0.78125" stopColor="white" stopOpacity="0.6" />
+          <stop offset="1" stopColor="white" stopOpacity="0" />
+        </linearGradient>
+        <linearGradient id="paint1_linear_1022_31454" x1="20.9999" y1="30.0001" x2="41.8133" y2="34.6018" gradientUnits="userSpaceOnUse">
+          <stop stopColor="white" />
+          <stop offset="1" stopColor="#FDE7E7" />
+        </linearGradient>
+        <clipPath id="clip0_1022_31454">
+          <rect width="60" height="60" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+  );
+}
+
+export const OOLogoIcon = ({ style = {}, ...rest }) => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 60 60" style={{ width: 60, height: 60, ...style }} {...rest}>
+      <circle cx="30" cy="30" r="30" fill="#FF4A4A" />
+      <rect width="38.1353" height="21.7041" x="10.9309" y="19.1437" fill="#FF4A4A" stroke="#fff" strokeWidth="2.1" rx="10.8521" />
+      <circle cx="22.4749" cy="29.8186" r="5.4425" stroke="#fff" strokeWidth="2.14286" />
+      <circle cx="37.5296" cy="29.8186" r="5.4425" stroke="#fff" strokeWidth="2.14286" />
+    </svg>
+  );
+}
+
+export const CheckmarkFilledIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElement = (props) => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 15 15" width={15} height={15} {...props}>
+      <circle cx="7.5" cy="7.5" r="7.5" fill="#272528" />
+      <path fill="#272528" d="M5 7.58063 6.9355 10 10 5" />
+      <path stroke="#fff" strokeLinecap="round" strokeLinejoin="round" strokeWidth="1.4" d="M5 7.58063 6.9355 10 10 5" />
+    </svg>
+  );
+}
+
+export const CallPutOptionsIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElement = (props) => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 60 60" width={15} height={15} {...props}>
+      <circle cx="30" cy="30" r="30" fill="#FF4A4A" />
+      <path stroke="#fff" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2.1" d="m13.0369 42.6843 9.5485-11.2845c1.6102-1.903 4.5814-1.7811 6.0304.2474 1.4706 2.0589 4.4995 2.1474 6.0878.1778l11.2965-14.0076" />
+    </svg>
+  );
+}
+
+export const LspIcon: (props: React.SVGProps<SVGSVGElement>) => React.ReactElement = (props) => {
```

Comment:
can u explain why you would use this pattern for the icons rather than just svgs as assets

Reply:
I use to write React components for small, lightweight and pure SVG icons/symbols. Stored images as assets require to be downloaded which may cause a small delay until they are rendered on the screen if they were not cached previously. This is annoying for small icons displayed in buttons and other elements